src/main/java/uk/nhs/hee/tis/trainee/forms/config/MongoConfiguration.java

```diff
@@ -54,5 +57,8 @@ public void initIndexes() {
     IndexOperations formRpartBindexOps = template.indexOps(FormRPartB.class);
     formRpartBindexOps.ensureIndex(new Index().on("traineeTisId", Sort.Direction.ASC));
     formRpartBindexOps.ensureIndex(new Index().on("lifecycleState", Sort.Direction.ASC));
+
+    IndexOperations ltftFormIndexOps = template.indexOps(LtftForm.class);
+    ltftFormIndexOps.ensureIndex(new Index().on("traineeId", Sort.Direction.ASC));
```

Comment:
to be moved to personalDetails.id

Reply:
I think we should experiment with moving the index creation to annotations anyway, it worked pretty well for CctCalculations in trainee-details